src/main/java/io/javalin/core/validation/Validator.kt

```diff
@@ -20,16 +20,24 @@ open class Validator<T>(val value: T, val messagePrefix: String = "Value") {
         return this
     }
 
-    fun get(): T = rules.find { !it.test.invoke(value) }?.let { throw BadRequestResponse(it.invalidMessage) } ?: value
+    fun get(): T = orElseNull() ?: throw BadRequestResponse("$messagePrefix cannot be null or empty")
+
+    fun orElseNull(): T? = rules
```

Comment:
Maybe `getOrNull` would be a more fitting method name? I see Kotlin is using that for collections: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/get-or-null.html

Reply:
No argument here.  Done.